grievance_social_protection/apps.py

```diff
@@ -18,6 +18,7 @@
     "gql_mutation_update_tickets_perms": ["127002"],
     "gql_mutation_delete_tickets_perms": ["127003"],
     "gql_mutation_create_comment_perms": ["127005"],
+    "gql_mutation_resolve_grievance_perms": ["127006"],
```

Comment:
Make sure this perm is listed on https://openimis.atlassian.net/wiki/spaces/OP/pages/1775173658/openIMIS+Authorities